src/helix/endpoints/schedule/create_channel_stream_schedule_segment.rs

```diff
@@ -42,6 +43,7 @@
 //! ```rust, no_run
 //! use twitch_api::helix::{self, schedule::create_channel_stream_schedule_segment};
 //! # use twitch_api::client;
+//! # use std::convert::TryFrom;
```

Comment:
if the crate is bumped to 2021 edition this would not be needed, for now it is though since it's on 2018

Reply:
Ah, good point. I was wondering about that but forgot to check the edition.